KeenClient/KeenLogSink.h

```diff
@@ -0,0 +1,37 @@
+//
+//  KeenLogger.h
+//  KeenClient
+//
+//  Created by Brian Baumhover on 2/16/17.
+//  Copyright © 2017 Keen Labs. All rights reserved.
+//
+
+// A given log level will include all
+// messages in lower verbosity levels.
+// For example, if the log level is set
+// to KLL_WARN, messages will log level
+// of both KLL_WARN and KeenLogLevelError will
+// be logged.
+typedef NS_ENUM(NSInteger, KeenLogLevel) {
```

Comment:
Is there guidance in Obj-C land around providing a null object here? It could be useful to have a log level of None == 0.